docs/sphinx/sections/api/apidocs/libraries/dagster-looker.rst

```diff
@@ -0,0 +1,6 @@
+Looker (dagster-looker)
+-----------------------------------------------------
+
+.. TODO description
```

Comment:
Might be worth putting off the apidoc addition until we add `version.py` so the module is published same time as the docs - otherwise we should fill out a desc here